Extractor/ExposedRoutesExtractor.php

```diff
@@ -104,14 +104,26 @@ public function getHost()
     {
         $requestContext = $this->router->getContext();
 
-        $host = $requestContext->getHost();
+        $host = $requestContext->getHost() .
+            ('' === $this->getPortExtension() ? $this->getPortExtension() : ':' . $this->getPortExtension());
```

Comment:
the colon should be added here instead I believe

https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/pull/320/files#diff-0993cf33f09a40107008a6597f40c4f6R123

Like it was done here also:

https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/pull/185/files#diff-0993cf33f09a40107008a6597f40c4f6R122

Otherwise this comment is wrong: https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/pull/320/files#diff-1955b791bdfe6c6e2f87f9dd89d24413R53

Reply:
I remove the colon `:` because the tests are wrong.

* For me, `portextension` should only contain port number without colon.
* In the JS test case , we need to test with `portextension: ':443'` instead of `portextension: 443`

Reply:
Why is it called port*Extension* then? can be just `port`?

Reply:
Yes, good question.
This came from the PR #185

Reply:
SO, I can rename `portExtension` by `port`. I think, like you said, everyone will know what it is ?